OnlineShopUI/online-shop-ui/src/components/Basket/Basket.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './Basket.scss';
+import Remove from '../../img/btn-remove.svg';
+import CloseCart from '../../img/btn-close.svg';
+
+const Basket = ({ onClose, opened }) => {
+  return (
+    <div className={opened ? 'overlay' : 'overlayUnVisible'}>
+      <div className="basket">
+        <div className="basket__title">
+          <h2>
+            Shopping cart
+            <button className="basket__cartClose" onClick={onClose}>
+              <img src={CloseCart} alt="CloseCart" />
+            </button>
```

Comment:
From UX perspective it's better to use the same closing pictures for all modals. Let's change it to be the same as in login modal